routing/pathfind.go

```diff
@@ -817,23 +818,28 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 		// blob.
 		var payloadSize uint64
 		if fromVertex != source {
+			// In case we do not have a payload size function
+			// available we skip this edge.
+			if edge.hopPayloadSizeFn == nil {
+				log.Errorf("skipping edge=%v unable to "+
```

Comment:
Why not `log.Criticalf` here? If we haven't set a payload function on an edge then we have a bug, no?

Reply:
> why shutting down the node if a payment fails (or just an edge in the pathfinding is somehow corrupt), because we could still have some htlcs to be resolved 

I think that generally we should shut down if we _know_ something is wrong with the code. Fail loudly so that it's noticed and fixed quickly rather than have something wrong fly under the radar.

Reply:
agree with this reasoning, will change the `Critical`